src/controllers/usersController.js

```diff
@@ -0,0 +1,107 @@
+const {
+  createUser,
+  getUserByID,
+  updateUserByID,
+  deleteUserByID
+} = require('../models/usersModel');
+const { inputValidation } = require("../utils/tools.js");
+const { ContentTypeError, PropNullorEmptyError, PropRequiredError } = require("../utils/errors.js");
+
+const postUsers = async (req, res, next) => {
+
+  let success = true;
+  try {
+    // content type
+    if (!inputValidation.checkContentType(req, res)) throw new ContentTypeError();
+    // all required attributes are provided
+    let errList = inputValidation.getMissingAttrs(res, req.body, ["firstName", "lastName", "email", "password", "zipCode"]);
```

Comment:
I see that we are removing `session ID` and `Size` in the other PR. Please make sure it is removed after the other PR is merged.